tests/smoke/test_cli_smoke.py

```diff
@@ -0,0 +1,62 @@
+"""Smoke tests for wristpy cli."""
+
+import pathlib
+
+import pytest_mock
+
+from wristpy.core import cli, config, orchestrator
+
+
+def test_main_default(
```

Comment:
This is now, strictly, testing the function as a unit. I.e. this need not be in the smoke directory anymore.

Reply:
Okay, moved main tests to cli unit tests.